src/assets/data/merchList.json

```diff
@@ -1,9 +1,33 @@
-[{
+[
+    {
+        "name": "Winter 2023 Merch",
+        "path": "winter-2023",
+        "description": "Unisex, ultra-cotton black shirts by Gildan with a screen-printed 'design' and thick, durable 2 in. x 2 in. vinyl stickers.",
+        "inStock": true,
+        "buyLink": "https://forms.gle/TJNv9zixcZehMBXq5",
+        "items": [
+            {
+                "name": "UC Irvine Embroidered T-Shirt",
```

Comment:
This old name doesn't quite match the new item which is a screen-printed, long-sleeve shirt

Reply:
good catch